gloom/srv/config.json

```diff
@@ -0,0 +1,10 @@
+ {
+  "sendgrid_api_key": "pleasechangeme",
+  "datadog_api_key": "pleasechangeme",
+  "datadog_app_key": "pleasechangeme",
+  "sentry_url": "pleasechangeme",
+  "datadog_ntp_server": "uk.pool.ntp.org",
+  "send_logs": true,
+  "send_stats": true,
+  "production": true
+ }
```

Comment:
This file would be best integrated within `config.py` if possible. See `config-example.py`.